gateways/paypalpayflow/request_builder.go

```diff
@@ -0,0 +1,87 @@
+package paypalpayflow
+
+import (
+	"fmt"
+
+	"github.com/BoltApp/sleet"
+)
+
+var (
+	defaultVerbosity    string = "HIGH"
+	defaultTender       string = "C"
+	defaultMIT          string = "MIT"
+	MITUnscheduled      string = "MITR"
+	CITUnscheduled      string = "CITU"
+	CITInitial          string = "CITI"
+	CITInitialRecurring string = "CITR"
+	MITRecurring        string = "MITR"
+)
+
+func buildAuthorizeParams(request *sleet.AuthorizationRequest) *Request {
+	expirationDate := fmt.Sprintf("%02d%02d", request.CreditCard.ExpirationMonth, request.CreditCard.ExpirationYear%100)
+	amount := sleet.AmountToDecimalString(&request.Amount)
+	var CARDONFILE *string = nil
```

Comment:
All caps usually means something is a static const. Can we make these local vars by making them camelCase like the other vars? For CARDONFILE and all of the billing address fields.